ClientApp/viking-vault/src/components/SideBar.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import './styles.css'
+import dashboard from './images/dashboard.png'
+import sagetutza from './images/arrowGRIdeschis.png'
+import placeholder from './images/placeholder.png'
+import paymentmethod from './images/payment-method.png';
+import block from './images/password.png'
+import transfer from './images/transfer.png'
+import card from './images/business-card-of-a-man-with-contact-info.png'
+
+interface ISideBarState{
+    show: boolean;
+}
+
+class SideBar extends React.Component<any, ISideBarState> {
+    constructor(props:any){
+        super(props)
+        
+        this.state = {
+            show: true
+        } 
+        this.clickHandler = this.clickHandler.bind(this)
+    }
+
+    clickHandler(event: any){
+        this.setState (oldState => {
+           return {
+               show: !oldState.show
+           }
+        })
+    }
+
+    render(){
+        return(
+            <div className={"sideBar-" + this.state.show} id="mySidebar">
+                <div className="regularDiv">
+                    <p> <img src={dashboard} alt=""/> My Dashboard  <img src={sagetutza} style={this.state.show ? {transform:'none'} : {transform:'scaleX(-1)'} } alt="" onClick={this.clickHandler}/></p>
+                </div>
+                <div className = {"sideMenu-" + this.state.show}>
```

Comment:
You can maybe extract this value ``"sideMenu-" + this.state.show`` and store it as a variable inside of render and just use that variable in the classNames in the JSX elements